context/app/static/js/components/workspaces/style.js

```diff
@@ -8,4 +8,11 @@ const StyledButton = styled(WhiteBackgroundIconButton)`
   height: 36px;
 `;
 
-export { StyledButton };
+const Left = styled.div`
+  flex-shrink: 0;
```

Comment:
What does this look like without `flex-shrink: 0`? Struggling to visualize the impact.

Reply:
Actually, plain divs work; Replaced.